modules/cookbook/pages/recipe-onboarding.adoc

```diff
@@ -0,0 +1,122 @@
+
+This recipe shows you how to onboard a single client, for a recipe about onboarding multiple clients at the same time, see xref:FIXME[].
+
+
+
+== Solution
+
+When you have completed this recipe, your client is onboarded, and it can be seen in the systems list of the {productname} {webui}.
+You can then use {productname} to manage the client.
+
+
+
+== Before you begin
+
+Before you start, you should already have:
+
+* A {productname} Server installed, which you can access using the {webui}.
+* A client machine with an operating system installed, which you can access across the network that your {productname} Server is on, using SSH.
+* Appropriate subscriptions from http://scc.suse.com for the products you are using.
+
+This recipe uses a {sles} 15 SP2 operating system.
+You can use other Linux operating systems, but some of the steps might be different.
+For more information on onboarding other clients, see xref:client-configuration:registration-methods.adoc[].
+
+
+
+== Steps
+
+Configure a fully-qualified domain name (FQDN) on your client::
+
+* On the client, at the command prompt, show the current hostname:
++
+----
+hostname -f
+----
++
+This command will probably return an error, or show something like ``localhost``.
+* Set a new hostname.
+  Your new hostname should include two periods.
+  In this example, we are using ``client1.suma.example``
++
+----
+hostnamectl set-hostname client1.suma.example
+----
+* Check that your change was successful:
++
+----
+hostnamectl
+----
+* Open {yast} and navigate to menu:Network Services[Hostnames].
+Edit the hostname to match the one you just set, and click btn:[OK].
+* In {yast}, navigate to menu:System[Network Settings] and go to the [guimenu]``Hostname/DNS`` tab.
+In the [guimenu]``Static hostname`` field, type your new hostmame.
+* Check that the change was successful:
++
+----
+hostname -f
+----
++
+This command should return your new FQDN.
+
+Prepare software channels on the {productname} Server::
+
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard].
+* In the [guimenu]``Organization Credentials`` tab, ensure you have entered your {scc} credentials, and are correctly authenticated.
+* In the [guimenu]``Products`` tab, ensure that the product catalog is fully updated:
++
+image::scc_sync.png[scaledwidth=80%]
+* Use the product search bar to find the channels you need for your client operating system.
+Check the channels you want to install, and click btn:[Add products]:
++
+image::add_products.png[scaledwidth=80%]
+* Wait for the products to fully synchronize.
+  Depending on the products you have chosen, this could take a long time.
+
+
+Create an activation key::
+
+* In the {productname} {webui}, navigate to menu:Systems[Activation Keys], and click btn:[Create Key].
+* Give your activation key a name, and select the base channel that matches the client you want to onboard.
+This should be the product you enabled in the last step:
```

Comment:
"procedure" instead of "step"?

Reply:
What about this:

```suggestion
This should be the product you just enabled:
```

Or, maybe take it out entirely?